Bio/Phylo/TreeConstruction.py

```diff
@@ -758,7 +838,11 @@ def _height_of(self, clade):
         if clade.is_terminal():
             height = clade.branch_length
         else:
-            height = height + max([self._height_of(c) for c in clade.clades])
+            for terminal in clade.get_terminals():
+                path = clade.get_path(target=terminal)
+                height = 0
+                for value in path:
+                    height = height + value.branch_length
```

Comment:
Style wise, that could be:

```python
height += value.branch_length
```

(The same suggestion could have been made to the original code too)

Reply:
Thanks for catching this!  Fixed with 670f2ae